lib/Interpreter/Interpreter.cpp

```diff
@@ -163,8 +171,15 @@ namespace cling {
     m_RawInputEnabled(false) {
 
     m_LLVMContext.reset(new llvm::LLVMContext);
+    if (!m_LLVMContext)
+      return;
     m_DyLibManager.reset(new DynamicLibraryManager(getOptions()));
+    if (!m_DyLibManager)
+      return;
+
     m_IncrParser.reset(new IncrementalParser(this, llvmdir));
+    if (!m_IncrParser || !m_IncrParser->isValid(false))
+      return;
```

Comment:
That is now a valid state, for output to work we need to support some level of the Interpreter returning from it's constructor gracefully when it wasn't fully initialized.